### ISSUE TITLE
Rstudio: CTRL+SHIFT+D option cannot run roclets

### ISSUE THREAD

**Issue by mikldk** [mikldk/roxytest#5]
Wish:
Rstudio: CTRL+SHIFT+D option to run `devtools::document()`/`roxygen2::roxygenise()` instead of `devtools::document(roclets=c('rd', 'collate', 'namespace'))`

* Project options -> Build tools -> If all check marks are removed, nothing happens with CTRL+SHIFT+D. If instead `devtools::document()` would be ran it would work.
* See <https://github.com/rstudio/rstudio/issues/5201>

**Comment by mikldk** [comment]
See also https://github.com/rstudio/rstudio/issues/5201.

**Comment by mikldk** [comment]
Maybe this can be solved by an Rstudio add-in:

Add file `inst/rstudio/addins.dcf` with content like this (from <https://github.com/r-lib/devtools/blob/master/inst/rstudio/addins.dcf>):

```
Name: Report test coverage for a package
Description: Calculate and report the test coverage for the current package, using `devtools::test_coverage()`.
Binding: test_coverage
Interactive: false
```

**Comment by mikldk** [comment]
See the proposed addin PR at https://github.com/r-lib/devtools/pull/2188 .

**Comment by mikldk** [comment]
PR was merged in r-lib/devtools@919b3f474a82dcaf9bcabc0c60919ea5a0a424f3.

**Comment by mikldk** [comment]
And is included in `devtools` 2.2.0: [see link](https://github.com/cran/devtools/blob/1caab804128e527ce5e662308c16c14734878395/inst/rstudio/addins.dcf#L16).

**Comment by mikldk** [comment]
For now we need to use `devtools`' addin and redefine keyboard shortcut.
